Rock/Model/CodeGenerated/DataViewService.cs

```diff
@@ -83,6 +84,55 @@ public bool CanDelete( DataView item, out string errorMessage )
             }  
             return true;
         }
+
+        /// <summary>
+        /// Create a new non-persisted Data View using an existing Data View as a template. 
+        /// </summary>
+        /// <param name="dataViewId">The identifier of a Data View to use as a template for the new Data View.</param>
+        /// <returns></returns>
+        public DataView GetNewFromTemplate(int dataViewId)
+        {
+            var item = this.Queryable()
+                           .AsNoTracking()
+                           .Include( x => x.DataViewFilter )
+                           .FirstOrDefault( x => x.Id == dataViewId );
+
+            if (item == null)
```

Comment:
@dcs619 Perhaps you can help me here. On the topic of minor amendments to PRs, what is the best method for submitting changes like this? In the past, I have tried simply pushing a new commit to the branch in my forked repo, but this doesn't squash the commits in the PR, just adds a second commit (which is not preferred as I understand it). Is there any way to do this easily, because otherwise it requires me to close and re-create the PR (which is time-consuming)?